credentials/single/single.go

```diff
@@ -0,0 +1,42 @@
+// Package single implements a basic credentials helper with a single username and secret.
+package single
+
+import (
+	"github.com/docker/docker-credential-helpers/credentials"
+	"github.com/pkg/errors"
+)
+
+// AuthStore is a basic credentials store that holds a single entry.
+type AuthStore *Credential
+
+// Add sets the store's credentials.
+func (s AuthStore) Add(cred *credentials.Credentials) error {
```

Comment:
> ... but there is no shared interface defined...

The shared interface is [`credentials.Helper`][1].  But since all we really need is a getter, I can remove `Add`, `Delete`, and `List` here.

[1]: https://godoc.org/github.com/docker/docker-credential-helpers/credentials#Helper

Reply:
> But since all we really need is a getter, I can remove `Add`, `Delete`, and `List` here.

Done with 8ce2b8b -> 5e011a3.